src/main/java/com/unbxd/hummus/dao/DaoModule.java

```diff
@@ -0,0 +1,21 @@
+package com.unbxd.hummus.dao;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Provides;
+
+import java.util.Properties;
+
+
+public class DaoModule extends AbstractModule {
+    @Override
+    protected void configure() {
+
+        bind(SchemaDao.class).toProvider(DaoProvider.class).asEagerSingleton();
+    }
+
+    @Provides
+    public Properties getProperties() {
+
+        return System.getProperties();
```

Comment:
how are you passing the properties? Could you please some readme on how to deploy this service